README.md

```diff
@@ -73,6 +89,24 @@ This is the usual stream write function. The object must have the following keys
     'numeric-entities': true
 ```
 
+## Cover image
+
+The epub specification does not contain a standarized way to include book covers. There is however a "best practice" that will work in most reader applications. `streampub` has some magic under the hood to correctly add a cover image. The only requirements are that the file needs to be in JPEG format and should be max 1000x1000 pixels.
+
+### Example
+```
+var Streampub = require('./index')
+var fs = require('fs')
+
+var epub = new Streampub({title: 'My Example'})
+epub.setAuthor('Example author')
+epub.pipe(fs.createWriteStream('example.epub'), {end: false})
+epub.write({id: 'cover-image', sourceFileName: 'image.jpg'})
+epub.write({chapterName: 'Chapter 1', content: '<h1>Chapter 1</h1><b>doc content</b>'})
+epub.write({chapterName: 'Chapter 2', content: '<h1>Chapter 2</h1><b>doc content</b>'})
+epub.end()
+```
```

Comment:
Yeah me too! :) Maybe a separate example section in the `README.md` would be nice.